environment/README.md

```diff
@@ -20,6 +22,16 @@ _Lists the geological and physical environment details of collection points._
 | _Amplitude_ | Analysis Amplitude, in cm
 | _Lag_ | Analysis Phase Lag, in degrees
 
+#### _FEATURES_ ####
+
+| Field | Description |
+| --- | --- |
+| _Station_ | Code used to uniquely identify Recording _Station_.
```

Comment:
does this table also need to add a column for the site or will we rely on the `network/sites.csv` and `network/stations.csv` for that?

Reply:
technically the "site" doesn't exist, only the station and the locations at the station.